client/web/compose/src/views/Admin/Pages/Builder.vue

```diff
@@ -573,6 +594,10 @@ export default {
         this.toastErrorHandler(this.$t('notification:page.duplicateFailed'))
       }
     },
+
+    checkUnsavedBlocks (next) {
```

Comment:
I know we talked about not commenting code, but I think it would be beneficial if there was a comment here describing what this is for